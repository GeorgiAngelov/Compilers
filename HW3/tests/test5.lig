function main(args : [[int]]) : int {
      var x : int;
      var x : int;
      return (x);
}
