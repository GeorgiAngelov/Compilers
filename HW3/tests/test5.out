(decls
   (decl-fun:NONE (id main) (type-fun (params (typed-id (id args) (type-array (type-array (type-int))))) (type-int))
      (decls
         (decl-var:ok (id x) (type-int))
         (decl-var:NONE (id x) (type-int)))
      (stmts
         (return:ok (id:int x)))))
Typechecking failed!
